### DISCUSSION TITLE
Scrollbar missing after delete Folder/Object

### DISCUSSION THREAD

**Discussion by Thrandu** [dani-garcia/vaultwarden#3504] [Q&A]
If i delete a folder or a object the scrollbar is gone, i have to refresh the website and then its back.
Test it on 2 Installation, same issue.

**Comment by BlackDex** [comment]
First, this is probably a web-vault GUI issue. Which we do not maintain.

Second, do you have a screenshot and a bit more descriptive way on how to reproduce this?

**Reply by Thrandu** [reply]
Hey, sry for the minimalistic information, i thought that would be enought.
I did the same on 2 PCs on 2 different Vaultarden Installations.
Open Chrome -> Login -> All Entrys then there is my Scrollbar on the right.
If i go in a folder and delete the folder, the Scrollbar is gone and never comes back.

Maybe you can do it yourself, create a new folder, maybe Test...no entries, ofc there is no scrollbar.
Delete the Folder and go back to All Tresors and the Scrollbar is still gone.

**Reply by Thrandu** [reply]
Ok i think we can remove this, i did it on a third PC with another Installation and there it works, so it looks like 2 dif. PCs and dif. Installation have some issue, maybe if i find the solution i will response for everyone else.
